src/PrestaShopBundle/Resources/views/Admin/Sell/Order/Order/view.html.twig

```diff
@@ -39,13 +39,13 @@
   </div>
 
   <div class="row">
-    <div class="col-md-4">
+    <div class="col-md-4" style="order: 2">
```

Comment:
Are you sure about this, you are breaking the two columns layout with this And inverting their order as well

Reply:
this was added temporary, it allowed the QA team to review the feature, the mechanism for making the product block bigger is in another PR...